docs/docs/anatomy.md

```diff
@@ -0,0 +1,254 @@
+---
+sidebar_position: 3
+---
+
+# Anatomy of an Equinox Service
+
+An Equinox service is composed of a few modules (namespaces).
+
+- **Stream**: tells us which category the service writes to and how the identity
+  of the stream is composed
+- **Events**: tells us which events the service writes
+- **Fold**: tells us how the events are folded into state
+- **Decide**: tells us which actions can be taken upon the state (resulting in
+  new events being written)
+- **Service**: the class that wraps the above into a cohesive domain service.
```

Comment:
maybe a `Queries` section for rendering helpers and dto types like https://github.com/jet/equinox/pull/429 or is this too early?

Reply:
Definitely not too early, I'll add it and mention it as optional

Reply:
also maybe call out that the other things are private/pubternal and this is the thing that you talk to

Reply:
> pubternal

Only to be discussed at the pub, never in the office

Reply:
Query section added